Dockerfile.coreagent

```diff
@@ -21,7 +21,14 @@ RUN New-Item \"\azp\agent\" -ItemType directory | Out-Null; `
 
 RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1')); `
     choco install -y docker-cli dotnetcore-sdk dotnet-5.0-sdk jq azure-cli; `
-    choco install -y dotnet-sdk --pre;
+    choco install -y dotnet-sdk --pre; `
+    Write-Host \"Installing nodejs\"; `
+    choco install -y nodejs-lts; `
```

Comment:
I would suggest to add a third type like "vsceagent" or similar, same as we have bcagent and coreagent for BC and .net core respectively